docs/dev-tools/chopsticks/overview.md

```diff
@@ -265,56 +208,43 @@ Chopstick's internal WebSocket server has special endpoints that allow the manip
 
 These are the methods that can be invoked and their parameters:
 
-???+ function "**dev_newBlock** (newBlockParams) — Generates one or more new blocks"
+??? function "**dev_newBlock** (newBlockParams) — Generates one or more new blocks"
 
     === "Parameters"
 
         |        Name         |                                 Type                                  |                    Description                     |
```

Comment:
These should not be in tables anymore, they should be bulleted items like the Zombienet PR